src/components/Display.js

```diff
@@ -1,28 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-// eslint-disable-next-line react/prefer-stateless-function
-export default class Display extends React.Component {
-  // eslint-disable-next-line no-useless-constructor
-  constructor(props) {
-    super(props);
-  }
+const Display = ({ result }) => (
+  <>
+    <h1>
+      { result }
+    </h1>
+  </>
+);
 
-  render() {
-    const {
-      result,
-    } = this.props;
-
-    return (
-      <div>
-        <h1>
-          { result }
-        </h1>
-      </div>
-    );
-  }
-}
+Display.defaultProps = {
+  // eslint-disable-next-line react/default-props-match-prop-types
+  result: '0',
+};
 
 Display.propTypes = {
+  // eslint-disable-next-line react/require-default-props
```

Comment:
Let's remove the inline comment, you don't need it, you already added the default props.

In other words; 

With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇